### PR TITLE
Created the HELP command list to include the descriptions for each of the command that is available to the user.

### PR DESCRIPTION
**Pull request by AjeyAshok**
Resolves Issue #44

### CODE REVIEW

**Comment by AjeyAshok** [comment]
> Hey Ajey yea looks neat and we need to add a description for every new task we create?

Yes, from now on every new command needs to have a description so that it could be displayed to the user when the HELP command is entered.

File: src/main/java/executor/command/Command.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 public abstract class Command {
     protected Boolean exitRequest = false;
     protected CommandType commandType;
+    protected String description = "Basic";
```
Comment (Mudaafi) [inline src/main/java/executor/command/Command.java]: Maybe we can change it to "MISSING DESCRIPTION" so if any of us runs the HELP command its very clear its missing.

File: src/main/java/executor/command/CommandDelete.java [hunk 2]
```diff
@@ -7,9 +7,11 @@
 public class CommandDelete extends Command {
     protected String userInput;
 
+
```
Comment (Mudaafi) [inline src/main/java/executor/command/CommandDelete.java]: Random newline, please remove.

File: src/main/java/executor/command/CommandDelete.java [hunk 3]
```diff
@@ -7,9 +7,11 @@
...
     // Constructor
     public CommandDelete(String userInput) {
         this.userInput = userInput;
+        this.description = "Deletes the specific entry that the user wants to remove";
```
Comment (Mudaafi) [inline src/main/java/executor/command/CommandDelete.java]: Descriptions should ideally include the format which the user should input the command so they can immediately use it. Command Delete uses an index-based deletion so it should be something like Delete <index_of_task_to_delete>.

// Sample
Deletes a specific entry based on the index provided. (E.g "Delete 2" will delete the 2nd task on your task list)

File: src/main/java/executor/task/FixedDuration.java [hunk 4]
```diff
@@ -2,13 +2,15 @@
 
 public class FixedDuration extends Task {
 
+    String input;
```
Comment (Mudaafi) [inline src/main/java/executor/task/FixedDuration.java]: Please make the declaration explicit. If it's supposed to be private, do indicate.

Comment (Mudaafi) [reply src/main/java/executor/task/FixedDuration.java]: Also, is this property really necessary?

**Pull request merged**